eldritch/ancient_ones.py

```diff
@@ -0,0 +1,169 @@
+import abc
+
+from eldritch import events, places, characters, monsters, values
+
+
+class AncientOne(metaclass=abc.ABCMeta):
+  # pylint: disable=unused-argument
+  def __init__(self):
+    self.doom = 0
+    self.combat_difficulty = 0
+    self.attributes = []
+
+  def get_interrupts(self, owner, state):
```

Comment:
This should be named get_interrupt, though you should also be inheriting it from GlobalEffect.